aws/dms/tasks.yml

```diff
@@ -34,6 +34,7 @@ cron:
 - dashboard.contained_levels
 - dashboard.level_sources_multi_types
 - dashboard.paired_user_levels
+- dashboard.parental_permission_requests
```

Comment:
This table contains PII. Can we either exclude the PII columns from the DMS export, or provide justification in this PR or the Jira ticket for extracting the PII fields from the primary database?